portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -314,6 +314,37 @@ async function loadSearch() {
   insertSearch(otherElements);
 }
 
+/**
+ * Gets links from the navigation bar
+ * Removes link for current page we are on
+ * @return {array}: the links from other pages.
+ */
+function getOtherLinks() {
+  const pagePath = window.location.pathname;
+  const pathSize = pagePath.length;
+
+  let linkElements =
+      document.getElementById('navBar').getElementsByTagName('a');
+  linkElements = Array.from(linkElements);
+  let numLinks = linkElements.length;
+
+  for (let i = 0; i < numLinks; i++) {
+    const link = linkElements[i].href;
+    if (pathSize < 2) {
```

Comment:
Can we add a comment mentioning why pathSize < 2 makes a difference?

Reply:
Okay sure.